### ISSUE TITLE
Limit Request: pyqt5-tools - 4GB per release / I guess close to 30GB total uploaded

### ISSUE THREAD

**Issue by altendky** [pypa/pypi-support#675]
**Project** pyqt5-tools - https://pypi.org/project/pyqt5-tools/

**Size of release/project** 4GB per release / I guess close to 30GB total uploaded based on hitting the quota

**Which indexes** PyPI

**Reasons for the request**

I would like to get out at least one more release since a few people have been requesting 3.9 support.  After that, I am working through breaking the project up into multiple sub-projects.  This will allow the biggest bits (the Qt applications) to have one less matrix dimension (one build for all Python versions).  So that's a 4x or so reduction in total size per release.  I am also splitting out the second biggest piece, the PyQt plugins, to a separate project.  This doesn't make for reduced matrix axes for the plugins but it does mean I don't have to re-release both pieces for a change in just one.  The main wrappers will be available in a third project and are just pure Python code.  This bit may move to a new project name as PyQt is starting to work on Qt 6 support so the PyQt5 name is overly specific.

As I write this out, it sounds like a lot of 'just make new projects to bypass the quotas' but...  I guess the best I can do is link the first two out of three new builds as evidence I'm seriously trying to reduce my footprint.

https://github.com/altendky/pyqt5-tools/pull/61
https://github.com/altendky/pyqt5-tools/pull/66

I hesitate to pull old releases as they are going to be the hardest for people to recreate if they need them.  Since I'm making reasonably quick progress on the splitting effort, if you don't think it's appropriate to increase the limit I'll probably just wait until I get the split done to do the next release.

Thanks for all the resources you provide to me and so many other developers.

**Comment by di** [comment]
Hi @altendky, thanks for making an effort to reduce the footprint here.

Looking at the project, it has an upload limit of 100MB and a total project size limit of 30GB and currently uses 29.9GB.

Do you need an additional 30GB for the total size? Or a 4GB upload limit? It's not clear to me from your request.

**Comment by altendky** [comment]
My original request was to have the project limit increased significantly so I could quickly get out a 3.9 supporting release.  That does not matter anymore as I have completed the breakdown of the project.  But, I do still plan to continue releasing under this name so it would be nice to have some space available that isn't just the unknown rounding error between 29.9 and 30.  `:]`  The entire v3 release of pyqt5-tools was 137KB so even a project limit increase of 100MB would be more than sufficient for the foreseeable future.

The other projects (pyqt5-plugins, qt5-tools, and qt5-applications) will stay within the standard project limit forever, forever, and probably forever respectively.

tl;dr, potentially no changes are needed anymore but a minimal increase of 100MB, to 30.1GB, for the project limit of pyqt5-tools would be a nice buffer.

Thanks for you time and consideration.

**Comment by di** [comment]
I've set the project limit for `pyqt5-tools` to 35 GB on PyPI.
